### ISSUE TITLE
Add automated a11y testing to CI

### ISSUE THREAD

**Issue by thatbudakguy** [sul-dlss/earthworks#1057]
See prior work on other repos, e.g. https://github.com/sul-dlss/pre-assembly/pull/1475

**Comment by dnoneill** [comment]
Created tests for geoblacklight instance but realized that due to the bl8 branch not working we can't run the tests against that and the tests on main reveal errors that are no longer errors in blacklight 8 so it is not worth putting on main.

**Comment by dnoneill** [comment]
closed with https://github.com/sul-dlss/earthworks/pull/1268
